sigs/orchestration-security/README.md

```diff
@@ -0,0 +1,15 @@
+# Orchestration Security SIG
+
+## Join us!
+- The Orchestration Security SIG is open to the public and recurs every other Monday at 10AM PT: https://docker.zoom.us/j/181832520
+- The living Google Doc where all notes are taken in real-tine: https://docs.google.com/document/d/1co6Jv9Mq8jeToK-sYNNXwUQiPWcDCvlNJ5bozAOfriE
+- Moby project forum topic: https://forums.mobyproject.org/c/sig/orchestration-security
+- #orchestration-sec channel on [Docker Community Slack](https://dockercommunity.slack.com/) (sign up [here](https://community.docker.com/registrations/groups/4316)
```

Comment:
I think you need an extra `)`: `(sign up [here])((https://community.docker.com/registrations/groups/4316)